tapestry.php

```diff
@@ -11,7 +11,8 @@
  */
 
 // Used to force-refresh assets
-$TAPESTRY_VERSION_NUMBER = '2.41.0-beta';
+$TAPESTRY_VERSION_NUMBER = '2.44.0-beta';
+add_option( "tapestry_version", $TAPESTRY_VERSION_NUMBER );
```

Comment:
Change the current version so the update kicks in. Also rename to tapestry_plugin_version so it's clear. 
```suggestion
add_option( "tapestry_plugin_version", "2.41.0-beta" );
```
I also suggest moving this to the update file itself.